charts/vela-core/templates/webhook.yaml

```diff
@@ -8,6 +8,50 @@ metadata:
   annotations:
     cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ .Values.certificate.certificateName }}
 webhooks:
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: {{ template "kubevela.name" . }}-webhook
+        namespace: {{ .Release.Namespace }}
+        path: /mutating-core-oam-dev-v1alpha2-applicationconfigurations
+    failurePolicy: Fail
+    name: mappconfig.kb.io
```

Comment:
I think we can use `mutating-core-oam-dev-v1alpha2-applicationconfigurations` as the name.

Reply:
Got it.